### ISSUE TITLE
Create files relative from the actual position

### ISSUE THREAD

**Issue by eskimoblood** [ritwickdey/vscode-create-file-folder#1]
It would be nice if I could create files relative to the actual position in the explorer

**Comment by ritwickdey** [comment]
hi, Can you explain further with an example for better understanding?

**Comment by eskimoblood** [comment]
At the moment the file is created in the root of the project. But it would be cool if I can created on the position where the folder that is selected in the explorer.

**Comment by ritwickdey** [comment]
I got your point! Thanks for the suggestion!

**Comment by ritwickdey** [comment]
**A Tips:** Just right click on a folder from explorer window and choose `create new file` - the relative path from workspace will be auto filled.

> Watch the gif (2nd part)

![./vscode-create-file-folder/img/screenshots/screenshots1.gif](https://github.com/ritwickdey/vscode-create-file-folder/raw/master/img/screenshots/screenshots1.gif)

**Comment by tiagojdf** [comment]
Thanks for this great plugin. Really like it.
Regarding this issue, it would be great if the shortcut would do that as well.

**Comment by elitan** [comment]
Yes, it would be great if the shortcut would do that as well. This is especially good if you, like me, want to avoid touching the mouse/trackpad when coding. 😎

@ritwickdey Any updates on this? How could we help you with this?

**Comment by OmranAbazid** [comment]
Also, one more thing would be useful, is to be able to create a file in the same directory of the currently open file.

**Comment by joshmu** [comment]
I would love to also see this! As a VIM user I don't use the mouse/trackpad, so a keyboard shortcut to create files relative to current open file would be amazingly useful. =)
